standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/cache/SharedCache.java

```diff
@@ -2476,6 +2499,25 @@ public boolean isTableConstraintValid(String catName, String dbName, String tblN
     return isValid;
   }
 
+  public boolean isTableCacheStale(String catName, String dbName, String tblName, String validWriteIdList) {
+    boolean isValid = false;
```

Comment:
nit: rename the variable to `isStale` instead? Below we are setting isValid to true when the the writeIdList in CachedStore is stale/older than client.

Reply:
Done